### ISSUE TITLE
Not getting credentials back after 1st connection

### ISSUE THREAD

**Issue by mrTuomoK** [CliffS/ikea-tradfri#8]
Hi. After a long time I now updated to this new 4.0.0 and having a bit of a trouble getting it working. I am able to connect for the first time always but not getting the credentials to save back.

I added logging the credentials inside the .finally(() => {} in the ikea-tradfri's connect() and the *this.credentials* seems fine (an object containing the two properties)

but on my app side if I log the credentials after connect: 

tradfri.connect().then((credentials) => { console.log('credential: ', credentials); });

I get *Symbol(connected)* So I don't seem to get the credentials back somehow, any ideas?

**Comment by philipodev** [comment]
Hi. I had the same problem.
What instead worked was reading `credentials` from the `Tradfri` object instead of the promise response. 

So instead of the example use something like this (javascript):
```javascript
try {

        const tradfri = new Tradfri(GATEWAY_IP, identity);
        await tradfri.connect();
        
        console.info("credentials", tradfri.credentials);  // this gives u correct credentials

    } catch(err){
        console.error(err);
    }
```

**Comment by mrTuomoK** [comment]
Thanks, I'll give it a shot :)

**Comment by CliffS** [comment]
Hi guys.  Sorry I've not responded sooner.  I'll fix this annoying bug as soon as I can.

**Comment by CliffS** [comment]
Fixed (I believe) in v4.0.1.

Thanks for using this library.

**Comment by CliffS** [comment]
As I've heard nothing more, I'm assuming this is now working again.  Please let me know if there are further problems.
